ansible/roles/ceph-node-exporter/templates/node_exporter.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Node Exporter
+
+[Service]
+User=node_exporter
+EnvironmentFile=/etc/{{ sysconfig_dir }}/node_exporter
+ExecStart=/usr/local/sbin/node_exporter $OPTIONS
```

Comment:
Hmm, `/usr/local/`? This will break on centos/rhel.

Reply:
It actually won't, since the rpm ships its own service file; when deploying outside of `devel_mode` we'll skip this.